graphs/fire-throttle.fbp

```diff
@@ -0,0 +1,23 @@
+# fire-throttle.fbp
+
+# Listens for a callback and fires the throttle component to send the next data 
+
+# Exports
+INPORT=accept.LIMIT:LIMIT
+INPORT=accept.ENCODING:ENCODING
+INPORT=accept.TYPE:TYPE
+INPORT=webserver.LISTEN:LISTEN
+INPORT=throttle.THROTTLE_SIZE:THROTTLE_SIZE
+OUTPORT=throttle.OUTPUT:OUTPUT
+
+# When a request comes in sent it to the accept node and respond
+webserver(webserver/Server) REQUEST -> INPUT accept(rdf-components/http-accept)
+accept ACCEPTED -> IN response(webserver/SendResponse)
+accept REJECTED -> IN response
+
+# Send whatever was received from the webserver as input to throttle-fire
+accept(rdf-components/http-accept) OUTPUT -> INPUT throttle(rdf-components/throttle-fire)
+'1' -> THROTTLE_SIZE throttle(rdf-components/throttle-fire)
+
+# Error handling echos errors to console.log
+accept ERROR -> IN error(core/Output)
```

Comment:
Not a whole lot that can actually go wrong, but I'll put something in for that.  It's good to cover it.  Thanks.